### ISSUE TITLE
Catch interpolation-only expressions

### ISSUE THREAD

**Issue by Charles546** [terraform-linters/tflint#525]
Since tf 0.12.14, tf spits out warnings about deprecating interpolation-only expressions.  It is an easy fix to existing tf files, but we need some kind of linter to catch future offenders in our CI pipelines.  It would be nice to have this feature in `tflint`.

**Comment by Charles546** [comment]
Below is an example of the expression and the warning message.

```
resource "google_storage_bucket" "foo" {
  name          = "bar"
  location      = "${var.location}"
  storage_class = "MULTI_REGIONAL"
}
```

Error message

```

Warning: Interpolation-only expressions are deprecated

  on gcs.tf line 44, in resource "google_storage_bucket" "foo":
  44:   location      = "${var.location}"

Terraform 0.11 and earlier required all non-constant expressions to be
provided via interpolation syntax, but this pattern is now deprecated. To
silence this warning, remove the "${ sequence from the start and the }"
sequence from the end of this expression, leaving just the inner expression.

Template interpolation syntax is still used to construct strings from
expressions when the template includes multiple interpolation sequences or a
mixture of literal strings and interpolations. This deprecation applies only
to templates that consist entirely of a single interpolation sequence.

```
